content.js

```diff
@@ -167,39 +167,52 @@ if (window.contentScriptInjected !== true) {
 
     function looksLikeForeignWord(word) {
         word = word.trim().toLowerCase();
+        if (word == "")
+            return false;
```

Comment:
It's good practice to use braces in if statement even if it has only one line block, see [this](https://eslint.org/docs/rules/curly). Also for the sake of consistency since somewhere you use it and then somewhere you don't.